test/integration/runonce_duration_test.go

```diff
@@ -0,0 +1,101 @@
+// +build integration
+
+package integration
```

Comment:
Could you have expressed these tests with a test table as with the unit tests? Not a big deal, just wondering for future reference.